### ISSUE TITLE
source editor does not show end of lines

### ISSUE THREAD

**Issue by franzlst** [DLR-RM/RAFCON#515]
If the source editor is too narrow for the the code line length, the user can scroll to the right. Unfortunately the last characters of the line are hidden.

Originally created by @sebastian-brunner  (Sebastian.Brunner@dlr.de) at 2018-04-19 12:53:06+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
It might be necessary to set a minimum width for the sidebar

Originally created by @franzlst  (Franz.Steinmetz@dlr.de) at 2018-05-02 11:37:37+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
So I checked how we can set the minimum sidebar. Easiest is to change the width request of the right side bar container by exchanging [line ](https://rmc-github-server/common/rafcon/blob/2b91cd4b1531938cbab8e6a96f536b9eb14260a8/source/rafcon/gui/glade/main_window.glade#L992)
`<property name="width_request">53</property>`
by 
`<property name="width_request">500</property>`

Generally this is somehow ugly. 

Other option is to adjust the the hscroller of the source editor container accordingly the position of the paned or as depended to the source editor hscroller if focus is on the source editor or even anytime.

Originally created by @Rbelder  at 2018-05-16 09:31:20+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
OK the other option I described is not really possible.

Originally created by @Rbelder  at 2018-05-16 10:17:17+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
Respective pull-request is merged and therefore the issue can be closed.

Originally created by @Rbelder  at 2018-05-25 15:02:10+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
not fixed yet, see
https://wiki.robotic.dlr.de/RAFCON/Meetings#Next_meeting

Originally created by @sebastian-brunner  (Sebastian.Brunner@dlr.de) at 2018-05-28 13:39:47+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
So the half solution to the issue was released now and the quit final solution can be seen in the develop branch sorry that it is mixed into the other stuff we wanted to release and I branched to late from the develop and badly checkout, merged and pushed the develop. In the release is the pane adaptation which jumps in one case.

The current implementation still includes the pane motion (happens only while cursor motion), see f676f35, but also use the spacer-frame (works on expose event) we voted for in the meeting, see  295c31c (the commit is not verified by me because it was badly cherry-picked by me). Both features are integrated to not clash with each other.

Originally created by @Rbelder  at 2018-05-30 14:01:33+00:00 (moved from RMC internal repository)
